frontend/web/src/styles/tokens.css

```
@@ -1,33 +1,88 @@
+    --color-slate-100: #1e293b;
+    --color-slate-200: #334155;
+    --color-slate-300: #475569;
+    --color-slate-400: #64748b;
+    --color-slate-500: #94a3b8;
+    --color-slate-700: #e2e8f0;
+    --color-slate-800: #f1f5f9;
+    --color-white: #0b1220;
+    --color-navy-900: #020617;
+    --color-navy-800: #0f172a;
+    --color-navy-700: #1e293b;
+    color-scheme: dark;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)